examples/helm-charts/cubestack/README.md

```diff
@@ -0,0 +1,76 @@
+# Cube Stack Chart
+
+This Helm Chart is an Umbrella chart wrapping everything that's needed to run an entire Cube Stack:
+
+- Cube Server
+- Cube Store
+- Redis
+
+## Add the Bitnami Helm Repo
+
+```bash
+helm repo add bitnami https://charts.bitnami.com/bitnami
+```
+
+## Update Dependencies
+
+```bash
+helm dependency update
+```
+
+## Install the Cube Stack
+
+Installing the Chart with the `values.yaml` file looks like this:
+
+```bash
+helm install cubestack . -f ./values.yaml
+```
+
+Because this is an Umbrella Chart, you can set any value from the dependent charts in the `values.yaml`.
+
+Any value from the `../cubejs/values.yaml`, `../cubestore/values.yaml`, and `bitnami/redis` can be set in the `./values.yaml`.
+
+The requirement is to prefix the values with `cubejs`, `cubestore`, and `redis` respectively.
+
+Here's an example:
+
+```yaml
+# values.yaml
+
+cubejs:
+...
+
+  global:
+    ## The port for a Cube.js deployment to listen to API connections on
+    ##
+    apiPort: 4000
+
+    ## If true, enables development mode.
+    ##
+    devMode: false
+
+  redis:
+    ## The host URL for a Redis server
+    ## Naming this release "cubestack" will give you this default Redis URL 
+    url: redis://cubestack-redis-master:6379
```

Comment:
Maybe we can automatically configure this value if `redis.enabled` = true

Reply:
Do you have an idea on how to do this?